src/main/resources/application-test.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=${DB_URL}
+spring.datasource.username=${DB_USERNAME}
+spring.datasource.password=${DB_PASSWORD}
```

Comment:
Where are the properties `${DB_URL}` , `${DB_USERNAME}`, and `${DB_PASSWORD}` set?